Add total cpu percentages as a metric (#12525)
DataDog/integrations-core: system_core/README.md

Need help? Contact [Datadog support][9].

### Windows and high numbers of processors

Due to [the way Windows splits processors into groups][10], metrics
for individual cores collected from this integration may have invalid
values for Windows hosts with high (> 64) numbers of cores for a
portion of the cores.

Note that `*.total` metrics should still reflect accurate values in
that situation and only per-core metrics are affected.

> Users with this type of configuration are advised to set up the
> [Windows Performance Counters integration][11] to track counters
> inside the `Processor Information` in order to get accurate per-core
> metrics.

Reviewer: ```suggestion
Datadog recommends that users with this type of configuration set up the
[Windows Performance Counters integration][11] to track counters
inside the `Processor Information`. This enables users to get accurate per-core
metrics.
```
